beast/observationmodel/noisemodel/toothpick.py

```diff
@@ -19,10 +19,10 @@
 
 from tqdm import tqdm
 
-from .noisemodel import NoiseModel
-from ..vega import Vega
+from beast.observationmodel.noisemodel.noisemodel import NoiseModel
```

Comment:
This one is painful to look at (it's right, but I wonder if it's a sign that the `from noisemodel import *` trick should be used in `beast/observanmodel/noisemodel/__init__.py`? (cc @karllark)

(not necessarily for this PR, but as a broader task)

Reply:
Using __init__.py is make this prettier is something to think about.  I agree, not for this PR.